package/MDAnalysis/__init__.py

```diff
@@ -164,6 +164,8 @@
 _MULTIFRAME_WRITERS = {}
 _PARSERS = {}
 _SELECTION_WRITERS = {}
+# Registry of TopologyAttributes
+_TOPOLOGY_ATTRS = {}
```

Comment:
So this thing is global. Who adds to this and when?

Reply:
it's like the _READERS and _PARSERS dicts, it just keeps track of any Classes that get defined

Reply:
Also when a user actually wants to add something like `color`? Also is it possible to add custom attrs with this? (No need if not just want to know)

Reply:
I just tested. This then means we can only add attributes for which we have defined classes.

Reply:
Yeah, so things are included as they are defined.  It's always been the case (with the new system) that for an Attribute to exist there needs to be a class behind it